src/mono/netcore/CoreFX.issues.rsp

```diff
@@ -5,176 +5,17 @@
 -notrait category=failing
 -notrait category=IgnoreForCI
 
-####################################################################
-##  Microsoft.VisualBasic.Core.Tests
-####################################################################
-
-# https://github.com/mono/mono/issues/14854
--nomethod Microsoft.VisualBasic.Tests.ErrObjectTests.Clear
-
-# Windows-only tests, incorrect platform guards (https://github.com/dotnet/corefx/pull/40365)
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.SaveSetting
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.DeleteSetting
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.GetAllSettings
--nomethod Microsoft.VisualBasic.Tests.InteractionTests.GetSetting
-
-####################################################################
-##  System.Collections.Specialized.Tests
-####################################################################
-
-# TODO: Wrong exception
-# https://github.com/mono/mono/issues/14858
--nomethod System.Collections.Specialized.Tests.NameValueCollectionCtorTests.Ctor_NegativeCapacity_ThrowsArgumentOutOfRangeException
-
-####################################################################
-##  System.Collections.Concurrent.Tests
-####################################################################
-
-# fails only with release mono
--nomethod System.Collections.Concurrent.Tests.ConcurrentQueueTests.ReferenceTypes_NulledAfterDequeue
-
-####################################################################
-##  System.Diagnostics.Tracing.Tests
-####################################################################
-
-# System.SR needs implemented.
-# https://github.com/mono/mono/issues/14907
--nomethod BasicEventSourceTests.TestsManifestNegative.Test_GenerateManifest_InvalidEventSources
-
-####################################################################
-##  System.Drawing.Common.Tests
-####################################################################
-
-# System.ArgumentException Arg_ObjObjEx
--nomethod System.Drawing.Imaging.Tests.ImageAttributesTests.SetColorMatrix_InvalidFlags_ThrowsArgumentException
```

Comment:
I removed this test from rsp file but it's still being skipped due to `Condition(s) not met: "GetIsDrawingSupported" `. I'm observing the same for plenty of other System.Drawing.Tests (at least, on macos).

Reply:
`brew install mono-libgdiplus` fixed `Condition(s) not met: "GetIsDrawingSupported"` and all skipped tests run and passed but this particular one still fails with `Arg_ObjObjEx`.

Reply:
Filed https://github.com/dotnet/runtime/issues/2337